index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css">
+  <title>Portfolio - Setup and Mobile First</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="toolbar">
+    <div class="container-2">
```

Comment:
- [ ] Great job so far. Please kindly fix the fixed width you have given your portfolio to make it responsive right up to the desktop version(768px). Nice work.
![Screenshot (1101)_LI](https://github.com/jcmunav63/mv_portfolio2/assets/65298048/beafa2ce-771d-4973-96ae-056d94088e7b)